packages/babel-plugin/tests/actions.test.ts

```diff
@@ -125,7 +125,7 @@ describe("action transpilation", () => {
       import { model, state } from "./src/model";
       const myAction = model.action(({
         state
-      }) => function myAction () {
```

Comment:
Just because it's not necessary. It means when we have 


```js
export default function () {

}
```

we can use the same code to transform the `FunctionDeclaration` to a `FunctionExpression`.

Would you rather keep it?

Reply:
No, was just curious